src/lib/y2storage/clients/inst_prepdisk.rb

```diff
@@ -51,6 +54,24 @@ def commit
         storage.rootprefix = Yast::Installation.destdir
         storage.calculate_actiongraph
         storage.commit
+
+        mount_in_target("/dev", "devtmpfs", "-t devtmpfs")
+        mount_in_target("/proc", "proc", "-t proc")
+        mount_in_target("/sys", "sysfs", "-t sysfs")
+      end
+
+      def mount_in_target(path, device, options)
+        storage = Y2Storage::StorageManager.instance
+        target_path = storage.prepend_rootprefix(path)
+
+        if !Yast::FileUtils.Exists(target_path)
```

Comment:
Is not enough with Ruby's `FileUtils`? At some point we would like to see `Yast::Fileutils` die.